client/src/pages/AddTaskPage.tsx

```diff
@@ -17,9 +18,21 @@ export const AddTaskPage: React.FC<RouteComponentProps> = ({ history, match }) =
 
   const [title, setTitle] = useState<string>('');
   const [description, setDescription] = useState<string>('');
+  const [ showToast, setShowToast ] = useState<boolean>(false);
+  const [ errorMessage, setErrorMessage ] = useState<string>('');
 
   const [createTaskMutation] = useOfflineMutation(createTask, mutationOptions.createTask);
 
+  const handleError = (error: any) => {
+    if (error.offline) {
+      error.watchOfflineChange();
```

Comment:
Again, this seems unnecessary unless we're actually looking to do something with the result (e.g. printing some message). I think we can just remove it. Same for anywhere else that uses this pattern

Reply:
Very good point. Previous app had snack bar that showed that changes were replicated. It will be cool to restore it if possible to have this in react